.pre-commit-config.yaml

```diff
@@ -13,6 +13,8 @@ repos:
     rev: 3.7.1
     hooks:
     -   id: flake8
+        additional_dependencies:
```

Comment:
I prefer `additional_dependencies: [flake8-typing-imports]`

Reply:
Do you still want this in here even if I'm not using the `if False` check?

Reply:
Seems that Travis can't install it for non-py3.6 environments :/

Reply:
ah right dang, I'll migrate this to azure pipelines then